nf_core/pipeline-template/{{cookiecutter.pipeline_slug}}/docs/configuration/local.md

```diff
@@ -11,10 +11,10 @@ First, install docker on your system: [Docker Installation Instructions](https:/
 
 Then, simply run the analysis pipeline:
 ```bash
-nextflow run nf-core/{{ cookiecutter.pipeline_name }} -profile docker --reads '<path to your reads>'
+nextflow run {{ cookiecutter.github_repo }} -profile docker --reads '<path to your reads>'
```

Comment:
As above - `{{ cookiecutter.github_repo }}` doesn't exist now (replaced with `nf-core/{{ cookiecutter.pipeline_name }}`), so please revert all of the changes in this file.